src/components/personlinje/Personlinje.tsx

```diff
@@ -20,34 +25,113 @@ const Separator = () => (
+                <Separator />
+                <Clipboard buttonLabel={formatMessage('ariaLabel.kopierFnr')}>
+                    <Normaltekst tag="span">{props.sak.fnr}</Normaltekst>
+                </Clipboard>
+                <Separator />
+                <span className={styles.saksnummer}>
+                    <Normaltekst tag="span">{formatMessage('label.saksnummer')}&nbsp;</Normaltekst>
+                    <Clipboard buttonLabel={formatMessage('ariaLabel.kopierSaksnummer')}>
+                        <Normaltekst>{props.sak.saksnummer.toString()}</Normaltekst>
+                    </Clipboard>
+                </span>
+                {props.søker.sivilstand && (
+                    <span className={styles.sivilstandAndSeperator}>
+                        <Separator />
+                        <Sivilstand sivilstand={props.søker.sivilstand} />
+                    </span>
+                )}
+            </div>
 
-            <Separator />
+            <PersonAdvarsel person={props.søker} />
+        </div>
+    );
+};
 
-            <Clipboard buttonLabel={intl.formatMessage({ id: 'ariaLabel.kopierFnr' })}>
-                <Normaltekst tag="span">{props.sak.fnr}</Normaltekst>
-            </Clipboard>
+const Sivilstand = (props: { sivilstand: ISivilstand }) => {
+    const { formatMessage } = useI18n({ messages });
 
-            <Separator />
+    const [status, hentPerson] = useApiCall(fetchPerson);
 
-            <span className={styles.saksnummer}>
-                <Normaltekst tag="span">{intl.formatMessage({ id: 'label.saksnummer' })}&nbsp;</Normaltekst>
-                <Clipboard buttonLabel={intl.formatMessage({ id: 'ariaLabel.kopierSaksnummer' })}>
-                    <Normaltekst>{props.sak.saksnummer.toString()}</Normaltekst>
-                </Clipboard>
-            </span>
-            <PersonAdvarsel person={props.søker} />
-        </div>
+    React.useEffect(() => {
+        if (props.sivilstand.relatertVedSivilstand) {
+            hentPerson(props.sivilstand.relatertVedSivilstand);
+        }
+    }, []);
+
+    return (
+        <span className={styles.sivilstand}>
+            <Normaltekst tag="span">
+                {formatMessage('label.sivilstand')}: {formatSivilstandType(props.sivilstand.type, formatMessage)}
+            </Normaltekst>
+
+            {pipe(
+                status,
+                RemoteData.fold(
+                    () => null,
+                    () => <NavFrontendSpinner />,
+                    (err) => (
+                        <AlertStripe type="feil">
+                            {err?.statusCode === ErrorCode.Unauthorized
+                                ? formatMessage('feilmelding.ikkeTilgang')
+                                : err?.statusCode === ErrorCode.NotFound
+                                ? formatMessage('feilmelding.ikkeFunnet')
+                                : formatMessage('feilmelding.ukjent')}
+                        </AlertStripe>
```

Comment:
Er tanken att vi skal vise en error-melding oppe i person-linja? 🤔

Reply:
ja, vi må gjøre et personsøk for å hente navnet til EPS